crates/sui-rpc-loadgen/src/payload/get_object.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use futures::future::join_all;
+use sui_json_rpc_types::{SuiObjectDataOptions, SuiObjectResponse};
+use sui_sdk::SuiClient;
+use sui_types::base_types::ObjectID;
+
+use crate::payload::{GetObject, ProcessPayload, RpcCommandProcessor, SignerInfo};
+use async_trait::async_trait;
+
+#[async_trait]
+impl<'a> ProcessPayload<'a, &'a GetObject> for RpcCommandProcessor {
+    async fn process(&'a self, op: &'a GetObject, _signer_info: &Option<SignerInfo>) -> Result<()> {
+        let clients = self.get_clients().await?;
+
+        let mut tasks = Vec::new();
+        for object_id in &op.object_ids {
```

Comment:
as synced offline, we need to make this into smaller chuncks

Reply:
refactored logic - this is handled in rpc_command_processor.prepare. Caller can adjust this as needed